package-lock.json

```diff
@@ -2427,6 +2427,7 @@
 				"@babel/runtime": "^7.0.0",
 				"@wordpress/api-fetch": "file:packages/api-fetch",
 				"@wordpress/data": "file:packages/data",
+				"@wordpress/deprecated": "file:packages/deprecated",
```

Comment:
This is the only line I expect. What's with all the others?

Reply:
I have no idea why they show up :(

Reply:
Trying again with `node_modules` removed.

Reply:
No change.

Reply:
```sh
gziolo$ npm --version
6.7.0
```

This is probably why.

Reply:
Okay, I can reproduce from master running the latest npm.

Since CI runs with the latest version, I think it's probably good to expect people to align.

https://github.com/WordPress/gutenberg/blob/ff9961bd69c34c0caf488276b79c163bc63f94ff/.travis.yml#L33

Makes me wonder if we still need all the tedious processing of `package-lock.json` changes here:

https://github.com/WordPress/gutenberg/blob/master/bin/process-git-diff.js

Reply:
https://github.com/npm/npm/issues/17722 is still opened 🤷‍♂️

Reply:
I don't think they use their GitHub repository anymore.

Reply:
https://npm.community/t/package-lock-json-keeps-changing-between-platforms-and-runs/1129/16 - I found it in its new place. It wasn't easy.